components/centos/centos-openshift-setup/README.rst

```diff
@@ -1,10 +1,12 @@
-Vagrantfile: CentOS Docker Base
-===============================
+Vagrantfile: OpenShift Setup In ADB Vagrant Box
+===============================================
```

Comment:
Would like to see the heading as

Vagrantfile: OpenShift setup in ADB Vagrant box